vmdb/app/apis/miqservices_ops.rb

```diff
@@ -244,6 +243,7 @@ def agent_config(hostId, config)
 
   def agent_register(xmlFile)
     ret = false
+    require 'miq-xml'
```

Comment:
Yeah, I'm fearful that by doing this it's overkill.  Alternatively, you can just load miq-xml up front in boot.rb :stuck_out_tongue: I'm not sure what the right answer is.